registration/authorize_test.go

```diff
@@ -122,3 +123,69 @@ func TestAuthorizeNode(t *testing.T) {
 		})
 	}
 }
+
+func TestAuthorizeNodeCommon_DuplicateStore(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	fileStorage, err := storeonce.New(ctx)
```

Comment:
Nit: it's not file storage :-) It only matters in the sense that someone looking at this won't be like "oh, it's missing a cleanup"...